internal/driver/driver_test.go

```diff
@@ -203,6 +203,57 @@ func TestParse(t *testing.T) {
 	}
 }
 
+func TestAppendComment(t *testing.T) {
+
+	// Encode profile into a protobuf and decode it again.
+	protoTempFile, err := ioutil.TempFile("", "profile_proto")
+	if err != nil {
+		t.Errorf("cannot create tempfile: %v", err)
+	}
+	defer os.Remove(protoTempFile.Name())
+	defer protoTempFile.Close()
+
+	testFlagSet := baseFlags()
+	testFlagSet.bools["proto"] = true
+	testFlagSet.strings["comment"] = "test comment please ignore"
+	testFlagSet.strings["output"] = protoTempFile.Name()
+	testFlagSet.args = []string{"cpu"}
+
+	options := setDefaults(nil)
+	options.Flagset = testFlagSet
+	options.Fetch = testFetcher{}
+	options.Sym = testSymbolizer{}
+
+	if err := PProf(options); err != nil {
+		t.Errorf("error in PProf %v", err)
+	}
+
+	actualBytes, err := ioutil.ReadFile(protoTempFile.Name())
```

Comment:
actual -> got

Reply:
any reason to prefer want/got vs expected/actual? most testing frameworks i'm familiar with use expected/actual.